inc/packages/namespace.php

```diff
@@ -171,14 +171,19 @@ function install_plugin( string $id, ?string $version = null, $skin ) {
  * @return MetadataDocument|WP_Error
  */
 function fetch_metadata_doc( string $url ) {
-	$response = wp_remote_get( $url, [
-		'headers' => [
-			'Accept' => sprintf( '%s;q=1.0, application/json;q=0.8', CONTENT_TYPE ),
-		],
-		'timeout' => 7,
-	] );
-	if ( is_wp_error( $response ) ) {
-		return $response;
+	$response = get_site_transient( md5( $url ) );
```

Comment:
We're doing `md5( $url )` twice here, I'm wondering if we could do that only once, and then maybe prefix it too so it's a bit more recognizable in the transient cache?